modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -152,3 +152,20 @@ If changes made to APIs are not visible right after you make them, you can refre
 == Duplicated External Objects
 
 Validating and synchronizing the external data source generates duplicate external objects that will not be used. Delete these new objects.
+
+== Missing Feature Error
+
+If installation of Anypoint API Community Manager shows a `Missing feaure` error, ensure digital experiences (previously known as communities) are enabled in your API Community Manager Salesforce organization.
+
+Example:
+
+----
+1. Lightning Community Themes(communityThemeDefinitions/acm_template.communityThemeDefinition) Missing feature
+Installing this package requires the following feature and its associated permissions: Lightning Community Themes
+2. Lightning Community Templates(communityTemplateDefinitions/NTO.communityTemplateDefinition) Missing feature
+Installing this package requires the following feature and its associated permissions: Lightning Community Templates
+3. Navigation Link Sets(navigationMenus/acm_template_Nav0.navigationMenu) Missing feature
+Installing this package requires the following feature and its associated permissions: Navigation Link Sets
+----
+
+To fix this error, follow the installation prerequisite steps in the section xref:installation-prerequisites.adoc#set-up-a-domain-with-enabled-communities[set up a domain with enabled communities].
```

Comment:
Since we are using the text as a title, title-style caps are OK:
```suggestion
To fix this error, follow the installation prerequisite steps in the section xref:installation-prerequisites.adoc#set-up-a-domain-with-enabled-communities[Set Up a Domain with Enabled Communities].
```